tests/lib/MerkleTree.spec.ts

```diff
@@ -0,0 +1,110 @@
+import Cryptography from '../../src/lib/Cryptography';
```

Comment:
Its good to use "baseUrl": "src/lib", in the compilerOptions property in tsconfig.json file to prevent relative path imports

Reply:
Hi Anmol, I wanted to use baseUrl, but I am not able to get Jasmine to resolve the paths properly after 30 min of trying. I have created an [issue](https://github.com/decentralized-identity/sidetree-core/issues/12), feel free to make it work.